src/cdk/text-field/autosize.ts

```diff
@@ -187,13 +187,15 @@ export class CdkTextareaAutosize implements AfterViewInit, DoCheck, OnDestroy {
     // Long placeholders that are wider than the textarea width may lead to a bigger scrollHeight
     // value. To ensure that the scrollHeight is not bigger than the content, the placeholders
     // need to be removed temporarily.
-    textarea.style.height = 'auto';
-    textarea.style.overflow = 'hidden';
+    textarea.classList.add('cdk-textarea-autosize-measuring');
     textarea.placeholder = '';
 
     // Use the scrollHeight to know how large the textarea *would* be if fit its entire value.
-    textarea.style.height = `${textarea.scrollHeight}px`;
-    textarea.style.overflow = '';
+    // We subtract 4 from the scrollHeight because of the 2px top and bottom padding. Having the
+    // extra 2px on either side seems to fix inaccurate measurements in Chrome.
+    textarea.style.height = `${textarea.scrollHeight - 4}px`;
```

Comment:
Phrase something like... "The cdk-textarea-autosize-measuring class includes a 2px padding to workaround an issue with Chrome, so we account for that extra space here by subtracting 4 (2px top + 2px bottom)."